CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Add backend for swift-log (#1164)
 - Specify CocoaPods version to ensure `swift_version` attribute works (#1167)
 - Simplify `DDLogFileManager` callbacks for archived log files (#1166)
+- Use setter to replace kvo for filelogger (#1180)
```

Comment:
This is in the wrong section. 3.7.0 has already been released. Please move it up into the 3.7.1 section.

Reply:
Oh, my mistake.